File: RA1/JAVA/EjemploHerencia/Snake.java
package EjemploHerencia;

public class Snake extends Animal{
    private boolean isVenom;

    public Snake(String name, Boolean isVenom){
        super(name);//llamar al constructor de la clase superior, es decir, de clase Animal
        this.isVenom = isVenom;
    }
    
    @Override
    public void makeSound(){
        System.out.println(this.name + " sisea: SSSS!!!");
    }
}

File: RA1/PYTHON/ejemplo_herencia/snake.py
#from <archivo> import <elemento<class>>
from typing import overload
from animal import Animal

class Snake(Animal):
    def __init__(self, name, is_venom):
        #opcion 1: 
        super().__init__(name)
        #opcion 2: Animal.__init__(name)
        self.is_venom = is_venom

    def make_sound(self):
        print(self.name + " sisea: SSSSS...")

    